src/compressor/QatAccel.h

```diff
@@ -16,19 +16,21 @@
 #define CEPH_QATACCEL_H
 
 #include <memory>
+#include <mutex>
+#include <condition_variable>
+#include <vector>
```

Comment:
please order them in alphabetical order.

Reply:
done.